doc/classes/NavigationServer2D.xml

```diff
@@ -303,6 +303,13 @@
 				Returns all created navigation map [RID]s on the NavigationServer. This returns both 2D and 3D created navigation maps as there is technically no distinction between them.
 			</description>
 		</method>
+		<method name="is_baking_navigation_polygon" qualifiers="const">
+			<return type="bool" />
+			<param index="0" name="navigation_polygon" type="NavigationPolygon" />
+			<description>
+				Returns [code]true[/code] when the provided [param navigation_polygon] is being baked on a background thread.
```

Comment:
I was being told to avoid punning as it makes it harder for translators to change.
```suggestion
				Returns [code]true[/code] when the provided navigation polygon is being baked on a background thread.
```